CONTRIBUTING.md

```diff
@@ -107,14 +107,17 @@ And then run the tests in another terminal tab:
 # run all tests
 pnpm test
 
-# run tests while passing other flags to sub-program
-pnpm test -- --coverage --my-other-flag
+# watch all tests
+pnpm test:watch
 
 # run tests for a specific package
-pnpm test packages/my-desired-package
+pnpm test:filter packages/my-desired-package
```

Comment:
This `:filter` suffix feels like a step back but I understand the reasoning behind it. It would be great if some bash shenanigans could bring back the previous `pnpm test packages/my-desired-package`. I tried some and it didn't work out. It is what it is. :D

Reply:
Agreed, will note down as a follow-up :)